theme/templates/base.html

```diff
@@ -40,7 +46,7 @@
               <a class="icon fa-slack" href="https://slack.kinto-storage.org/" title="Slack channel">Slack</a>
             </li>
             <li>
-              <a class="icon fa-rss" href="http://www.kinto-storage.org/archives.html" title="Latest articles about Kinto">Blog</a>
+              <a class="icon fa-rss" href="./archives.html" title="Latest articles about Kinto">Blog</a>

```

Comment:
they serve different purposes: in the archive you have all the articles whereas on the news they are paginated (but this of course depends on the theme)